### DISCUSSION TITLE
https://nextjs.org/ is down?

### DISCUSSION THREAD

**Discussion by taoqf** [vercel/next.js#25219] [Help]
### What version of Next.js are you using?

10.0.3

### What version of Node.js are you using?

v15

### What browser are you using?

chrome

### What operating system are you using?

Fedora33

### How are you deploying your application?

next start

### Describe the Bug

when I tried to open <https://nextjs.org/> it shows: 

```text
This site can’t be reached
nextjs.org refused to connect.
Try:

Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_REFUSED
```

### Expected Behavior

```text
This site can’t be reached
nextjs.org refused to connect.
Try:

Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_REFUSED
```

### To Reproduce

I don't know if the server is broken down or just because I am blocked from gfw.

**Comment by michael32693** [comment]
same issue
